src/ex4/server/clients/pokemon_client.js

```diff
@@ -1 +1,36 @@
-// The Pokemon Client (using axios) goes here
\ No newline at end of file
+// The Pokemon Client (using axios) goes here
+const axios = require('axios');
+
+module.exports = class PokemonClient {
+  constructor() {
+    this.API_URL = 'https://pokeapi.co/api/v2/pokemon/';
+  }
+
+  async fetchPokemons(ids) {
```

Comment:
you don't need both `fetchPokemons` and `fetchPokemon` - just use the plural one anywhere you want instead of having duplicate logic
EDIT: I see that you reused the logic of the singular form in the plural form, well done. in this I would still recommend having only one function - but it's your choice, it's ok to leave it as is